pom.xml

```diff
@@ -787,6 +787,16 @@
                 <artifactId>smallrye-common-function</artifactId>
                 <version>${version.io.smallrye.smallrye-common}</version>
             </dependency>
+            <dependency>
+                <groupId>io.smallrye.common</groupId>
+                <artifactId>smallrye-common-net</artifactId>
+                <version>${version.io.smallrye.smallrye-common}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.smallrye.common</groupId>
+                <artifactId>smallrye-common-os</artifactId>
+                <version>${version.io.smallrye.smallrye-common}</version>
+            </dependency>
```

Comment:
Why not replace all of these with a single bom import?

```xml
<dependency>
    <groupId>io.smallrye.common</groupId>
    <artifactId>smallrye-common-bom</artifactId>
    <version>${version.io.smallrye.smallrye-common}</version>
    <type>pom</type>
    <scope>import</scope>
</dependency>
```

Reply:
Oh - I suppose this comment belongs here: https://github.com/wildfly/wildfly-core/pull/6482

Reply:
although ... bring more without needing is also an argument against it ... so, it will be a balance between it and parent pom organization/simplicity

Reply:
Well, ok, I guess the correct approach is to declare the smallrye common bom into the dependency management section, instead of the dependencies section to avoid that issue. And declare what we need into the dependencies section instead

Reply:
Upps sorry, my mistake; it is already on the management dependencies section, so it will bring only what's needed. Forget my initial comment.